packages/mui-material-nextjs/src/v13-pagesRouter/pagesRouterV13Document.tsx

```diff
@@ -102,17 +102,20 @@ export async function documentGetInitialProps(
         const { styles } = extractCriticalToChunks(initialProps.html);
         return {
           ...initialProps,
-          emotionStyleTags: styles.map((style) =>
-            style.css.trim() ? (
```

Comment:
We don't need this check anymore?

Reply:
Brought it back. It must be a mistake from a merge conflict.